src/renderer/services/plugin-manager.js

```diff
@@ -743,13 +743,19 @@ class PluginManager {
         },
 
         setInterval (...args) {
```

Comment:
I think it's worth explicitly specifying the arguments here (`callback` and `interval`) since 2 arguments are always required anyway

Reply:
Done, using `method` instead of `callback` to prevent the `no-callback-literal` error in eslint.